server/app/models/ProgramModel.java

```diff
@@ -268,12 +277,19 @@ private ProgramModel setLocalizedConfirmationMessage(ProgramDefinition.Builder b
     if (localizedConfirmationMessage != null) {
       builder.setLocalizedConfirmationMessage(localizedConfirmationMessage);
     } else {
-      builder.setLocalizedConfirmationMessage(
-          LocalizedStrings.create(ImmutableMap.of(Locale.US, "")));
+      builder.setLocalizedConfirmationMessage(LocalizedStrings.withEmptyDefault());
     }
     return this;
   }
 
+  private void setLocalizedSummaryImageDescription(ProgramDefinition.Builder builder) {
+    if (localizedSummaryImageDescription != null) {
+      builder.setLocalizedSummaryImageDescription(Optional.of(localizedSummaryImageDescription));
+    } else {
+      builder.setLocalizedSummaryImageDescription(Optional.empty());
```

Comment:
question (non-blocking): Why use `Optional.empty` here and not `LocalizedStrings.withEmptyDefault()`?

Reply:
Yeah, it's a bit confusing and I'm open to suggestions. For `ProgramDefinition`:
- `Optional.empty()` represents not having a description
- `Optional.of(LocalizedStrings)` represents having a description

I couldn't do the same for `ProgramModel` because Ebean doesn't support having `Optional` fields. So `ProgramModel` uses:
-`null` represents not having a description
- non-null represents having a description

That's why this gets converted from `null` to `Optional.empty()`. This is admittedly awkward, I can definitely add a comment here but I'm also super open to a different data structure too!

Reply:
Ohhh that makes sense! I'll comment more on the other PR, but using `Optional` to represent that the field is in fact optional makes a lot of sense to me. I think a comment explaining the `null` value on `ProgramModel` would be really helpful!

Reply:
Great call, added docs/comments in two places in `ProgramModel`.